examples/advanced_behaviors/dispatch_event/dispatch_event_append.xml

```diff
@@ -0,0 +1 @@
+<text xmlns="https://hyperview.org/hyperview">Let it Be!</text>
```

Comment:
could you add some basic style on this element so that it renders nicely in the demo context (i.e. with margins)

Reply:
Added some styling tweaks. Hope it looks good now.

![Simulator Screen Shot - iPhone 6 - 2019-07-22 at 17 08 37](https://user-images.githubusercontent.com/6682655/61629759-7428fa00-aca3-11e9-8c08-3c309f263dd6.png)